os/include/arpa/inet.h

```diff
@@ -199,6 +211,9 @@ uint16_t htons(uint16_t hs);
 /**
  * @brief Convert the Internet host address from the IPv4 numbers-and-dots notation into binary form
  *
+ * @details @b #include <arpa/inet.h>\n
+ * SYSTEM CALL API\n
+ * POSIX API (refer to : http://pubs.opengroup.org/onlinepubs/9699919799/)
```

Comment:
please check if this API is POSIX or not.

Reply:
I found that just like below.

Header : <arpa/inet.h>

in_addr_t    inet_addr(const char *);
char        *inet_ntoa(struct in_addr);
const char  *inet_ntop(int, const void *restrict, char *restrict, socklen_t);
int          inet_pton(int, const char *restrict, void *restrict);

If it is not the POSIX API, please let me know. I will revise that.